ADR/ADR041-use-activerecord-encryption.md

```diff
@@ -0,0 +1,32 @@
+# ADR041: Use ActiveRecord encryption in Runner
```

Comment:
Is this ADR about whether we encrypt data at the application? or have we've already agreed that, and this is this specifically about implementation details and whether we use ActiveRecord encryption?

Reply:
Have reframed to be about encrypting data at application level, and that the way to do it would be via ActiveRecord Encryption.